ReflectionAnnotationRegEx/src/main/java/com/onto/javacourse/displayprivate/DisplayPrivate.java

```diff
@@ -0,0 +1,98 @@
+package com.onto.javacourse.displayprivate;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class to display private fields and methods of a given object using reflection.
+ */
+public class DisplayPrivate {
+  private static final Logger LOGGER = LoggerFactory.getLogger(DisplayPrivate.class);
+
+  /**
+   * Displays the private fields and methods of the given object.
+   *
+   * @param instance The object whose private fields and methods are to be displayed.
+   */
+  public static void displayPrivate(Object instance) {
+    if (instance == null) {
+      LOGGER.warn("Provided instance is null. Cannot display class details.");
+      return;
+    }
+    Class<?> instanceClass = instance.getClass();
+    LOGGER.info("Class: {}", instanceClass.getName());
+    LOGGER.info("");
+    LOGGER.info("Fields:");
+    Field[] declaredFields = instanceClass.getDeclaredFields();
+    for (Field field : declaredFields) {
```

Comment:
Isn't this logic similar to the other PR (for diplaying class information. Why not extract and reuse instead of implementing it twice?